java-sdk/radar-schemas-tools/src/main/java/org/radarcns/schema/specification/passive/PassiveDataTopic.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 King's College London and The Hyve
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.radarcns.schema.specification.passive;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.radarcns.catalogue.ProcessingState;
+import org.radarcns.schema.specification.AppDataTopic;
+
+import java.util.Objects;
+
+/**
+ * TODO.
+ */
+public class PassiveDataTopic extends AppDataTopic {
+    public enum RadarSourceDataTypes {
```

Comment:
again similar to PHQ8 enum: should these types be derived from the specification catalogue TYPE (otherwise do we have to define here again)?